src/frontends/arcs/handle_connection_spec_test.cc

```diff
@@ -120,25 +118,20 @@ TEST_P(GetAccessPathTest, GetAccessPathTest) {
 }
 
 // Just some different strings that could be used as particle specs.
-std::string sample_particle_spec_names[] = {
-    "ParticleSpec",
-    "PS",
-    "P1",
-    "P2"
-};
+std::string sample_particle_spec_names[] = {"ParticleSpec", "PS", "P1", "P2"};
 
 ProtoStringAndExpectedAccessPathPieces protos_and_access_path_info[] = {
-    { .textproto =
-        R"(name: "handle_spec" direction: READS type: { primitive: TEXT })",
-      .handle_connection_name = "handle_spec",
-      .access_path_selectors_tostrings = { "" } },
-    { .textproto = R"(
+    {.textproto =
+         R"(name: "handle_spec" direction: READS type: { primitive: TEXT })",
```

Comment:
Also this global should be made `static const`.

Reply:
done